cli/tools/lint.rs

```diff
@@ -147,6 +149,12 @@ pub async fn lint(flags: Flags, lint_flags: LintFlags) -> Result<(), AnyError> {
   };
 
   let operation = |paths: Vec<PathBuf>| async {
+    let incremental_cache = Arc::new(IncrementalCache::new(
+      &ps.dir.lint_incremental_cache_db_file_path(),
+      // use a hash of the rule names in order to bust the cache
+      &lint_rules.iter().map(|r| r.code()).collect::<Vec<_>>(),
+      &paths,
+    )?);
```

Comment:
Oh, this doesn't work for linting for some reason. My guess would be the lint rules order is not stable and so I will sort it.

Reply:
Fixed now.